### PR TITLE
ResourceGraph updates for 2021-03-01 API version

### PR DESCRIPTION
**Pull request by fenwickt**
- Generated contracts for 2021-03-01 api-version from resourcegraph.json with the generate_sdk script. Had to change the readme file locally to add the -track2 suffix to the swagger-to-sdk python repo and #Python section, please advise if it's ok
- Added tests for new tenant and MG scenarios
- Ran packaging_tools.change_log and pasted the output to change log file
- Updated version to 8.0.0

### CODE REVIEW

File: sdk/resources/azure-mgmt-resourcegraph/tests/recordings/test_mgmt_resourcegraph.test_resources_single_management_group_query.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+interactions:
+- request:
+    body: '{"query": "project id, tags, properties | limit 2"}'
```
Comment (dramuy) [inline sdk/resources/azure-mgmt-resourcegraph/tests/recordings/test_mgmt_resourcegraph.test_resources_single_management_group_query.yaml:3]: >{"query": "project id, tags, properties | limit 2"}' [](start = 11, length = 52)

need to include mg here.

File: sdk/resources/azure-mgmt-resourcegraph/tests/recordings/test_mgmt_resourcegraph.test_resources_multiple_management_groups_query.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+interactions:
+- request:
+    body: '{"query": "project id, tags, properties | limit 2"}'
```
Comment (dramuy) [inline sdk/resources/azure-mgmt-resourcegraph/tests/recordings/test_mgmt_resourcegraph.test_resources_multiple_management_groups_query.yaml:3]: >{"query": "project id, tags, properties | limit 2"}' [](start = 11, length = 52)

multiple MGs needs to be added here

**Comment by dramuy** [comment]
Hello, @00Kai0 @changlong-liu @msyyc @jsntcy could you please help review this and unblock us.

**Review by dramuy — APPROVED** [review]
:shipit:

**Comment by msyyc** [comment]
Hi @fenwickt If you want to release Python SDK, please make an  issue on https://portal.azure-devex-tools.com/tools/request-api-release. For mgmt python SDK, we do not permit others to merge PR to the main repo.

**Comment by dramuy** [comment]
@msyyc we have submitted the request for release. Please let us know if you need any additional details on this.

---
In reply to: [807880065](https://github.com/Azure/azure-sdk-for-python/pull/17387#issuecomment-807880065) [](ancestors = 807880065)

**Comment by msyyc** [comment]
> @msyyc Yuchao Yan FTE we have submitted the request for release. Please let us know if you need any additional details on this.
> 
> In reply to: [807880065](https://github.com/Azure/azure-sdk-for-python/pull/17387#issuecomment-807880065) [](ancestors = 807880065)

@zhangyan133 will help to release. Please pay attention to the comment of https://github.com/Azure/sdk-release-request/issues/1415

**Pull request closed without merging**